NineChronicles.Headless/BlockChainContext.cs

```diff
@@ -0,0 +1,23 @@
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Explorer.Interfaces;
+using Libplanet.Store;
+using Nekoyume.Action;
+using NCAction = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
+
+namespace NineChronicles.Headless
+{
+    public class BlockChainContext : IBlockChainContext<NCAction>
```

Comment:
What's the difference between `BlockChainContext` and `StandAloneContext`? There seems to be a lot of duplicate arguments.

Reply:
As you said, they have common features and `BlockChainContext` just act like view of `StandaloneContext`. Why I made `BlockChainContext` was that `Libplanet.Explorer.Queries.Query` needs it.

https://github.com/planetarium/libplanet-explorer/blob/553cbf05d61eff4ab4d5c408768136225dc8f5cf/Libplanet.Explorer/Queries/Query.cs#L23-L35